lib/sablon/content.rb

```diff
@@ -223,6 +240,15 @@ def process_readable(source, attributes)
         #
         [File.basename(name), source.read]
       end
+
+      # Convert centimeters or inches to Word specific emu format
+      def convert_to_emu(unit, value)
```

Comment:
Good call making this a separate method, it will make adding new units much easier in the future. :+1: I do some unit conversion elsewhere and I might end up consolidating all of that logic into a single place eventually but that is beyond the scope of this PR.